plugins/inventory/proxmox.py

```diff
@@ -275,32 +275,33 @@ def _get_session(self):
         return self.session
 
     def _get_auth(self):
-
         validate_certs = self.get_option('validate_certs')
 
         if validate_certs is False:
             from requests.packages.urllib3 import disable_warnings
             disable_warnings()
 
         if self.proxmox_password:
-
             credentials = urlencode({'username': self.proxmox_user, 'password': self.proxmox_password})
-
             a = self._get_session()
-
             ret = a.post('%s/api2/json/access/ticket' % self.proxmox_url, data=credentials)
-
             json = ret.json()
-
             self.headers = {
                 # only required for POST/PUT/DELETE methods, which we are not using currently
                 # 'CSRFPreventionToken': json['data']['CSRFPreventionToken'],
                 'Cookie': 'PVEAuthCookie={0}'.format(json['data']['ticket'])
             }
-
         else:
+            # Clean and format token components
+            user = self.proxmox_user.strip()
+            token_id = self.proxmox_token_id.strip()
+            token_secret = self.proxmox_token_secret.strip()
+
+            # Build token string without newlines
+            token = f'{user}!{token_id}={token_secret}'.strip()
```

Comment:
This is not necessary, since you already stripped all parts before:
```suggestion
            token = f'{user}!{token_id}={token_secret}'
```